authors/apps/authentication/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from authors.apps.profiles.models import Profile
+
+from .models import User
+
+@receiver(post_save, sender=User)
```

Comment:
Great work @alexxsanya Please fix this code-climate issue

Reply:
@cdvx this issue has been fixed. thanks for pointing this out.

Reply:
@cdvx  this was fixed.